tests/aws/services/s3/test_s3_partitions.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+
+from localstack.testing.pytest import markers
+from localstack.utils.strings import short_uid
+
+
+class TestS3Partitions:
+    # We only have access to the AWS partition, not CHINA/US-GOV/etc
+    @markers.aws.manual_setup_required
+    @pytest.mark.parametrize("region,partition", [("us-east-2", "aws"), ("cn-north-1", "aws-cn")])
+    def test_sns_notifications_in_different_partitions(
```

Comment:
We should skip this test in the s3 image run, as sns will not be available, like other notifications test, using this skip marker: `@pytest.mark.skipif(condition=TEST_S3_IMAGE, reason="SQS not enabled in S3 image")`